docs/en/advanced_config/gnss_degraded_or_denied_flight.md

```diff
@@ -0,0 +1,71 @@
+
+<Badge type="tip" text="main (planned for: PX4 v1.17)" /> <Badge type="warning" text="Experimental" />
+
+::: warning Experimental
+This is a new feature with limited real-world testing.
+It is intended for GNSS dropout scenarios (not pure GNSS-denied from takeoff), and requires that alternative velocity/position sensors are available.
+
+Please [share your related test logs](../getting_started/flight_reporting.md#sharing-the-log-files-for-review-by-px4-developers) to help us verify and harden it.
+:::
+
+PX4 is default-configured for outdoor flight with a reliable GNSS signal, but it can also be set up in "dead-reckoning mode" to more gracefully handle environments where GNSS is intermittently degraded or denied during flight.
+
+This section describes the configuration options, and the circumstances in which they should be used.
+
+## Overview
+
+PX4's EKF2 navigation has two modes for handling when GNSS data is determined to be unreliable:
+
+- **Automatic mode** (the default): Used for flying outdoors in environments where a GNSS signal is expected to be largely reliable.
+- **Dead-reckoning mode**: Recommended when you want to fly missions or other position controlled mode when there is intermittent GNSS loss, such as when flying under a bridge, from outdoors into an indoor setting, or when there is GNSS jamming (it is not suitable for pure-indoor use, as a GNSS signal is required before arming).
+
+::: info
+Dead-reckoning mode helps for both Fixed-Wing and Multicopter vehicles.
+MC vehicles benefits more because it can hover when transitioning between sensor regimes.
+FW needs continuous accurate velocity/position during the entire mission arc, making sensor transitions trickier.
+:::
+
+### Automatic Mode
+
+In Automatic mode the EKF2 resets if GNSS is lost and no other sources of position are available.
+This can result in a [position loss failsafe](../config/safety.md#position-gnss-loss-failsafe) and may trigger a shift into a mode that does not require global position, including stopping missions.
+
+This is desirable if the GNSS signal is likely to be recovered quickly and there are no mechanisms to estimate position when GNSS is unavailable.
+
+Use Automatic (default) when:
+
+- Flying in open sky with reliable GNSS throughout the mission.
+- You want the EKF to reset to GNSS when it becomes available again.
+- Operating in environments where GNSS is either good or completely unavailable (binary state).
+
+### Dead-Reckoning mode
+
+In Dead-reckoning mode, EKF2 stops fusing GNSS data when it becomes unreliable and prevents EKF2 resets, provided there are other sources of position or velocity data that can be fused.
+This ensures that the vehicle can continue flying missions and other position controlled modes when GNSS is lost.
+
+When GNSS is recovered the vehicle position estimate will be updated, which may cause jerky movements (in position controlled modes) if the estimate has drifted.
```

Comment:
```suggestion
When GNSS is recovered it will be fused with other measurements when tests indicate it can be trusted. This may cause jerky movements in position controlled modes if the estimate has drifted.
```